cross/cp210x/Makefile

```diff
@@ -0,0 +1,35 @@
+PKG_NAME = cp210x
+PKG_EXT = zip
+PKG_DIST_SITE = http://www.silabs.com/Support%20Documents/Software
+PKG_DIST_NAME = Linux_3.x.x_VCP_Driver_Source.zip
+PKG_DIST_FILE = cpx210x.zip
+PKG_DIR = cpx210x
+
+REQ_KERNEL = 1
+
+HOMEPAGE = http://www.silabs.com
+COMMENT  = CP210x USB to UART Bridge VCP Drivers
+LICENSE  = 
+
+POST_EXTRACT_TARGET = myPostExtract
+CONFIGURE_TARGET = nope
+COMPILE_TARGET = myCompile
+INSTALL_TARGET = myInstall
+
+include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: myPostExtract
+myPostExtract:
+	@cd $(WORK_DIR) ; \
+	mv Linux_3.x.x_VCP_Driver_Source $(PKG_DIR) ; \
+
+.PHONY: myCompile
+myCompile:
+	cp $(KERNEL_DIR)/drivers/usb/serial/cp210x.c $(WORK_DIR)/$(PKG_DIR)/cp210x.c
+	CPPFLAGS="" CFLAGS="" LDFLAGS="" \
+	make -C $(KERNEL_DIR) SUBDIRS=$(WORK_DIR)/$(PKG_DIR) modules
+
+.PHONY: myInstall
+myInstall:
+	mkdir -p $(STAGING_INSTALL_PREFIX)/modules
+	install -m 755 $(WORK_DIR)/$(PKG_DIR)/$(PKG_NAME).ko $(STAGING_INSTALL_PREFIX)/modules/
```

Comment:
With one minor variation this worked fine. The variation being the MAKE arguments, from what I can tell they need to be as they where (which was taken from LIRC I believe) instead of the flags provided by $(MAKE). At least, while it compiled directly with the provided code, it failed to compile when built as a dependency of Domoticz until I re-adjusted the command.